customization/customization.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+dir=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd ) ; readonly dir
+cd "${dir}/.."
+
+# Stage 1 sync - intermediate to the ./vendir folder
+set -x
+vendir sync --file vendir.s1.yml --lock-file vendir.s1.lock
+{ set +x; } 2>/dev/null
+
+# Copy cilium original files so we can do a diff later on.
+cp -a ./vendor/cilium ./vendor/cilium.orig
```

Comment:
would this work if we kept vendor as the original and apply the patches directly into the helm directory?

then you could skip the 2 stages, and the diffs and patches are always referenced to the helm directory, and not upstream